homeassistant/components/nanoleaf/__init__.py

```diff
@@ -22,11 +22,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except InvalidToken as err:
         raise ConfigEntryAuthFailed from err
 
```

Comment:
The timeout on `get_info` might be a little to high as its blocking startup when the device is offline 

```
2021-09-28 14:59:34 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: nanoleaf
2021-09-28 15:00:34 WARNING (MainThread) [homeassistant.bootstrap] Waiting on integrations to complete setup: nanoleaf
```

Reply:
`ClientConnectorError` is raised on my instance when the device is offline.

Reply:
I will add a sock_connect timeout and bump the library in another PR.